src/errorDictionary.js

```diff
@@ -0,0 +1,63 @@
+import fs from "fs";
+
+let dictionary;
+
+module.exports = {
+  init: function (config) {
+    if(!dictionary && config){
+
+      // check if file
+      if(typeof(config) === 'string' && fs.existsSync(config)){
+        try{
+          dictionary = JSON.parse(fs.readFileSync(config, 'utf8'));
+        }catch(e){
+          console.warn("Error loading error dictionary from file " + config + ". " + JSON.stringify(e));
+        }
+      }else{
+
+        if(typeof(config) === 'string' && config.startsWith("{")){
+          config = JSON.parse(config);
+        }
+
+        var client = config.protocol === "https:" || (typeof(config) === "string" && config.startsWith("https")) ? require("https") : require("http");
+
+        try{
+          client.get(config, function (response) {
```

Comment:
will this lib https://github.com/TestArmada/nightwatch-extra/blob/master/package.json#L103 make the http code more concise?